src/viking.js

```diff
@@ -1,11 +1,84 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack(){
+        return this.strength;
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage;
```

Comment:
otra manera mas común de hacer una sustracción es usar -= y en suma += lo vas a ver muy común de hoy en adelante, entonces puedes poner this.health -= damage